### ISSUE TITLE
Membersite course - Fold option + picture

### ISSUE THREAD

**Issue by ChristinaChristiansen** [Simplero/Roadmap#171]
When using Multiple Modules + lessons ... 
It would be nice to get the lessons to fold on all Modules - except the one you have actively chosen
+ show a picture under the module
![slet simplero](https://user-images.githubusercontent.com/26918070/32146581-c218675c-bcd9-11e7-9950-1a35e6326180.JPG)

My teaching is very visual so a picture on what you learn say more than the titel.

Christina

**Comment by teresa-slp** [comment]
Yes, please!   I have a large 11-week course and the side bar navigation is way too long!

For more detailed comment and photos, please see post on 20 February, "Menu options in Membership sites"
